modules/client/src/addresslistVoting/internal/utils.ts

```diff
@@ -166,19 +166,41 @@ export function computeProposalStatus(
     parseInt(proposal.startDate) * 1000,
   );
   const endDate = new Date(parseInt(proposal.endDate) * 1000);
+  // The proposal is executed so the status becomes EXECUTED
+  // independently of the other conditions
   if (proposal.executed) {
     return ProposalStatus.EXECUTED;
   }
+  // The proposal is not executed and the start date is in the future
+  // so the status becomes PENDING
   if (startDate >= now) {
     return ProposalStatus.PENDING;
   }
-  if (proposal.potentiallyExecutable || proposal.earlyExecutable) {
-    return ProposalStatus.SUCCEEDED;
+  // The proposal is not executed and the start date is in the past
+  // So we must check if the proposal reached the approval threshold
+  // If it reached the approval threshold and it's a signaling proposal
+  // the status becomes SUCCEEDED
+  // If it reached the approval threshold and it's not a signaling proposal
+  // the status becomes SUCCEEDED if the end date is in the past or if the
+  // proposal is early executable
+  if (proposal.approvalReached) {
+    if(proposal.isSignaling) {
```

Comment:
```suggestion
    if(proposal.isSignalling) {
```
Wrong spelling, please fix everywhere.